generators/app/templates/app/controllers/apiInfo.js

```diff
@@ -0,0 +1,22 @@
+const { merge } = require('lodash');
+const pjson = require('../../package.json');
+
+exports.apiInfo = (req, res, next) => {
+  try {
+    return res.status(200).send(res.headers);
```

Comment:
Instead of get info of API from response headers, could we simply generate that info here ?

Reply:
i'd delete this file